src/components/Pagination/Pagination.tsx

```diff
@@ -76,7 +75,7 @@ const Pagination = ({
       </div>
 
       <IconButton
-        iconColor={'darkGray'}
+        color={'neutralBlack-700'}
```

Comment:
optionally you can avoid curly brakets here.
just `color="neutralBlack-700"`

Reply:
Sure! 😊